core/src/test/java/org/kohsuke/stapler/ClassDescriptorTest.java

```diff
@@ -119,15 +117,15 @@ public void overridingMethod() throws Exception {
         assertEquals(3, f.getAnnotation(AnnA.class).value());
         assertNotNull(f.getAnnotation(AnnB.class));
         // similarly parameter annotations should be fused together
-        assertSame(Nullable.class, f.getParameterAnnotations()[0][0].annotationType());
+        assertSame(AnnC.class, f.getParameterAnnotations()[0][0].annotationType());
 
         // method should be dispatched to D.x() which overrides B.x()
         assertEquals(2, f.bindAndInvoke(new D(), null, null, "Hello"));
     }
 
     public static class B<T> {
         @AnnA @AnnB
-        public int x(@Nullable T t) { return 1; }
+        public int x(@AnnC T t) { return 1; }
```

Comment:
The JSR 305 version was `RUNTIME` retention whereas the SpotBugs version is `CLASS` retention.